src/views/WorkspaceDetail.vue

```diff
@@ -207,7 +217,8 @@ export default defineComponent({
     const tables = computed(() => store.getters.tables);
     const networks = computed(() => store.getters.networks);
     const uploads = computed(() => store.state.uploads.filter(
-      (upload) => upload.status === 'PENDING' || upload.status === 'STARTED',
+      // Find uploads with PENDING, STARTED, or FAILED status and that are less than 15 minutes old
+      (upload) => (upload.status === 'PENDING' || upload.status === 'STARTED' || upload.status === 'FAILED') && (new Date().getTime() - new Date(upload.created).getTime() < 15 * 60 * 1000),
```

Comment:
What's the rationale behind the 15 minute cutoff?

Reply:
We don't save any state on the browser at the moment. This seemed like a reasonable duration to show the message for. Is there a better way to handle this, maybe using local storage?